fenix/app/src/main/java/org/mozilla/fenix/shopping/ui/ProductAnalysis.kt

```diff
@@ -467,6 +472,23 @@ private fun ProductRecommendation(
                         url = product.imageUrl,
                         modifier = Modifier.size(productRecommendationImageSize),
                         targetSize = productRecommendationImageSize,
+                        placeholder = {
+                            Box(
+                                modifier = Modifier
+                                    .size(productRecommendationImageSize)
+                                    .background(
+                                        color = FirefoxTheme.colors.layer3,
+                                        shape = RoundedCornerShape(8.dp),
+                                    ),
+                            ) {
+                                Image(
+                                    imageVector = ImageVector.vectorResource(id = R.drawable.ic_file_type_image),
```

Comment:
I'm curious if there was a specific reason to use this over `painterResource`, the docs mention:
```
This function is intended to be used for when low-level ImageVector-specific functionality is required. 
For simply displaying onscreen, the vector/bitmap-agnostic painterResource is recommended instead.
```

Reply:
I thought it would have a certain impact when scaling the vector drawable, but upon reading the [doc](https://developer.android.com/jetpack/compose/graphics/images/compare#image-vector) more carefully, it seems that it is mostly used when other customizations are needed. Changed back to painterResource.